### ISSUE TITLE
Move over interactable alerts so they aren't blocked by HUD buttons

### ISSUE THREAD

**Issue by Moonwards** [moonwards1/Moonwards-Virtual-Moon#735]
They should probably now go in the middle of the bottom of the screen.

![Screenshot from 2020-11-03 09-33-52.png](https://app.gitkraken.com/api/glo/boards/5e96433deb480d00123f5d87/attachments/5fa178e26ddf520011325cfa)

**Comment by Zach777** [comment]
The Interactable popup is a bit behind the times. I think I should make it where it says press [x] to interact with [interactable name] when you have a RayCast active. It would say Press [x] for Interacts Menu when it is in the Rover (or other Area using Interactors).

 It would take a bit more work though so I want your permission before I handle it.

**Comment by Zach777** [comment]
@Moonwards It would be nice to have an answer to the above question for what you want me to do.

**Comment by Moonwards** [comment]
Hm.

Having one line instead of two would be much better. 
I might say 'Press [x] to [y]` instead, which would mean the words there have to be specified in each case, but it would make for shorter, clearer instructions.

It seems like a good plan.
